tests/provision.fmf

```diff
@@ -0,0 +1,28 @@
+standard-inventory-qcow2:
+  qemu:
+    m: 2048
+    drive:
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
+      - size: 1073741824
+        interface: scsi
```

Comment:
do you need all of these for the snapshot tests?

Reply:
They are small (1 G) - but I could reduce the count if you'd like.

Block devices can't be shared across VGs unless they are partitioned.  So many small devices is slightly more flexible than a few bigger.  I'm fine with changing if you'd like.

Reply:
Ok.  This is fine for now.  We can revisit later if it is a problem